File: filmy-app/app/src/main/java/rk/entertainment/filmy/data/models/configuration/ConfigImageData.java
package rk.entertainment.filmy.data.models.configuration;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class ConfigImageData {

    @SerializedName("base_url")
    private String baseUrl;

    @SerializedName("secure_base_url")
    private String secureBaseUrl;

    @SerializedName("backdrop_sizes")
    private List<String> backdropSizes = null;

    @SerializedName("logo_sizes")
    private List<String> logoSizes = null;

    @SerializedName("poster_sizes")
    private List<String> posterSizes = null;

    @SerializedName("profile_sizes")
    private List<String> profileSizes = null;

    @SerializedName("still_sizes")
    private List<String> stillSizes = null;


    public String getBaseUrl() {
        return baseUrl;
    }

    public void setBaseUrl(String baseUrl) {
        this.baseUrl = baseUrl;
    }

    public String getSecureBaseUrl() {
        return secureBaseUrl;
    }

    public void setSecureBaseUrl(String secureBaseUrl) {
        this.secureBaseUrl = secureBaseUrl;
    }

    public List<String> getBackdropSizes() {
        return backdropSizes;
    }

    public void setBackdropSizes(List<String> backdropSizes) {
        this.backdropSizes = backdropSizes;
    }

    public List<String> getLogoSizes() {
        return logoSizes;
    }

    public void setLogoSizes(List<String> logoSizes) {
        this.logoSizes = logoSizes;
    }

    public List<String> getPosterSizes() {
        return posterSizes;
    }

    public void setPosterSizes(List<String> posterSizes) {
        this.posterSizes = posterSizes;
    }

    public List<String> getProfileSizes() {
        return profileSizes;
    }

    public void setProfileSizes(List<String> profileSizes) {
        this.profileSizes = profileSizes;
    }

    public List<String> getStillSizes() {
        return stillSizes;
    }

    public void setStillSizes(List<String> stillSizes) {
        this.stillSizes = stillSizes;
    }

}

File: filmy-app/app/src/main/java/rk/entertainment/filmy/data/models/configuration/ConfigImageData.kt
package rk.entertainment.filmy.data.models.configuration

import com.google.gson.annotations.SerializedName

data class ConfigImageData(
        @SerializedName("base_url")
        var baseUrl: String? = null,

        @SerializedName("secure_base_url")
        var secureBaseUrl: String? = null,

        @SerializedName("backdrop_sizes")
        var backdropSizes: List<String>? = null,

        @SerializedName("logo_sizes")
        var logoSizes: List<String>? = null,

        @SerializedName("poster_sizes")
        var posterSizes: List<String>? = null,

        @SerializedName("profile_sizes")
        var profileSizes: List<String>? = null,

        @SerializedName("still_sizes")
        var stillSizes: List<String>? = null
)